lib/scenic/configuration.rb

```diff
@@ -0,0 +1,34 @@
+module Scenic
+  class Configuration
```

Comment:
I looked more into this and I can't find any nstances of gems relying on this configuration store directly. It very much seems like it's intended for applications to store data they pass off to other bits of code, not for gems to hook into.

The kicker for me is that our custom configuration provides typo protection while the rails custom config is essentially a Hashie::Mash that will respond to anything.